### PR TITLE
Integrate gutenberg-mobile release 1.68.0

### PR DESCRIPTION
**Pull request by guarani**
## Description
This PR incorporates the 1.68.0 release of gutenberg-mobile.
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4362

Release Submission Checklist

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by guarani** [comment]
Marking this ready for review.

**Review by twstokes — APPROVED** [review]
LGTM @guarani! 👍

File: RELEASE-NOTES.txt [hunk 1]
```diff
@@ -1,5 +1,9 @@
 18.9
 -----
+* [**] Block editor: Fix scroll update when typing in RichText component. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/4299]
```
Comment (guarani) [inline RELEASE-NOTES.txt]: I think we can change this to be clearer to non-devs:
> Fix issue where editor doesn't auto-scroll so you can see what is being typed

(Adapted from issue https://github.com/wordpress-mobile/gutenberg-mobile/issues/4069)

Comment (guarani) [reply RELEASE-NOTES.txt]: Wdyt @twstokes?

Comment (twstokes) [reply RELEASE-NOTES.txt]: Sounds good @guarani - good call. Updated.

**Pull request merged**